launch-script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+export TMPDIR=$XDG_CACHE_HOME/tmp
+
+MOZ_LAUNCHER="$MOZ_DIST_BIN/launch-script.sh"
+##
+## Enable Wayland backend?
+##
+if ! [ $MOZ_DISABLE_WAYLAND ] && [ "$WAYLAND_DISPLAY" ]; then
+  if [ "$XDG_CURRENT_DESKTOP" == "GNOME" ]; then
+    export MOZ_ENABLE_WAYLAND=1
+  fi
+##  Enable Wayland on KDE/Sway
+##
+  if [ "$XDG_SESSION_TYPE" == "wayland" ]; then
+    export MOZ_ENABLE_WAYLAND=1
+  fi
+fi
```

Comment:
Just leave it at default if it isn't enabled OOTB. People can add the env var as an override.

Reply:
id like to keep it this way because it crashes firefox if not